cardano-cli/src/Cardano/CLI/Shelley/Run/Transaction.hs

```diff
@@ -44,7 +46,8 @@ import           Cardano.Api.TxSubmit as Api
 import           Cardano.Api.Typed as Api
 
 data ShelleyTxCmdError
-  = ShelleyTxCmdAesonDecodeProtocolParamsError !FilePath !Text
+  = ShelleyTxAesonDecodeMultiSigScriptError !FilePath !Text
```

Comment:
For consistency:

```suggestion
  = ShelleyTxAesonCmdDecodeMultiSigScriptError !FilePath !Text
```

Reply:
Yeah this got reversed in the rebase :+1: